Harvest.Jobs.Core/JobBase.cs

```diff
@@ -8,7 +8,7 @@ namespace Harvest.Jobs.Core
 {
     public abstract class JobBase
     {
-        public static IConfigurationRoot Configuration { get; set; }
+        public static IConfigurationRoot Configuration { get; set; } = null!;

```

Comment:
I don't really like that just to hide a warning. But I'll leave it to @srkirkland for a 🧑‍⚖️  (judgement)